puppet/lib/puppet/indirector/facts/puppetdb.rb

```diff
@@ -62,7 +62,7 @@ def find(request)
             Http.multi_arity_get(http_instance,
```

Comment:
@mullr Did y'all make the decision that the next release of puppetdb off of the master branch will require at least Puppet Server 5 to be present?  We had only added the `Http.multi_arity_get` function because we wanted to ensure that this code would work across both Puppet Server 2.x and 5.x (given the underlying arity difference across versions in the http_client get method) but if we no longer have to support Puppet Server 2.x from this code, we could just code to the new arity and make this code a bit simpler.